runelite-client/src/main/resources/npc_health.json

```diff
@@ -1160,5 +1160,11 @@
 	"Alchemical Hydra_426": 1100,
 	"Undead Druid_105": 140,
 	"Temple Spider_75": 70,
-	"Sarachnis_318": 400
+	"Sarachnis_318": 400,
+	"Basilisk Youngling_57" : 60,
+	"Monstrous Basilisk_135" : 170,
+	"Typhor_194" : 280,
+	"The Jormungand_363" : 600,
+	"Basilisk Knight_204" : 300,
+	"Basilisk Sentinel_351" : 460
```

Comment:
I don't think this is right https://oldschool.runescape.wiki/w/Basilisk_Sentinel

Reply:
It looks like he was buffed since my commit. I'll fix this.